### DISCUSSION TITLE
Light Mode - Menu Bar Disappears After Update

### DISCUSSION THREAD

**Discussion by Serendibb** [burkybang/SuiteAdvanced-Feedback#180] [General]
Just updated to v30.1.0 and suddenly all my menu "dissappeared". I am using Thorium browser and had flush the cached and restarted the machine without fixing it. If I disable the plugin, menu returns. Also, if I turn the settings to Dark Mode, I can make out the menu. Anybody else having the same issue?
![image](https://github.com/user-attachments/assets/5b0ef9f2-2394-4d54-b4ec-ab2135531124)

**Comment by burkybang** [comment]
I started using a simpler method for theming that is pretty new. Chrome has supported it for a while.

Please ensure your browser is up to date. I'm hoping other Chromium browsers are not behind.

**Comment by Serendibb** [comment]
I'll keep an eye out. Currently, I'm using the most recent Thorium version. Thanks for the quick reply


Thanks.



Regards

[cid:521ed0fd-d032-49f3-86c3-551ce43bbe88]

**Reply by HeavenlyEntity** [reply]
Similar situation. Yet mine is Sidekick and they do not have this update yet.

**Reply by burkybang** [reply]
Okay, it sounds like I may need to revert that one change for now.

**Reply by HeavenlyEntity** [reply]
Thank you sir! Yeah I appreciate it. I know you're getting ready for SuiteWorld hah. Simple enough. Let us know when its been fixed. 🚀

**Reply by burkybang** [reply]
I submitted a new update. Should be out within the next hour.

**Reply by burkybang** [reply]
v30.1.1 is out now. Sorry about that everyone!

**Reply by burkybang** [reply]
Is everything back to normal for you both now?

**Reply by HeavenlyEntity** [reply]
It is looking good! Thank you sir!

**Comment by burkybang** [comment]
# Minimum Required Chromium Browser Versions
- Chrome: 123
- Edge: 123
- Opera: 109

If Chrome is refusing to update due to some error, simply reinstall it. You do not need to uninstall Chrome first. Reinstalling will force the update.
Download from Here: https://www.google.com/chrome

**Comment by HeavenlyEntity** [comment]
Is it possible to downgrade for browsers that do not have this update yet? Would be nice to have a version select field to select the version that is compatible with the browser instead of going by chromium itself. This would eliminate having issues like this, now I have to disable it because all the headers are broken.

**Reply by burkybang** [reply]
I'm going to work on reverting that one change

**Reply by burkybang** [reply]
I submitted a new update. Should be out within the next hour.

**Comment by burkybang** [comment]
@Serendibb @HeavenlyEntity 

I'm wondering if Thorium and Sidekick support this feature now.

I put together a quick sample of the feature working at the link below. 

Could you please test and let me know?

https://playcode.io/2289023

**Comment by Serendibb** [comment]
Hi Adam,

Is this what you're after?

[cid:f03b8fd6-bb31-48e5-85f5-ef3747a23a53]



Don't hesitate to reach out if you have any questions or need clarification. I'm always here to help.


Thanks.



Serendibb



Getting this email out of normal working hours? I am sending you this email at a time that works for me. I only expect you to respond when convenient for you. #balance


________________________________

**Reply by burkybang** [reply]
The image didn't come through. Can you please send it again?

**Comment by Serendibb** [comment]
[cid:f6413655-9de3-45c6-8371-09419eecaeb1]



Don't hesitate to reach out if you have any questions or need clarification. I'm always here to help.


Thanks.



Regards

[cid:536a20a7-bc82-4cc6-bee7-0f13e9f307ba]

Song Hymn Ooi
Consultant, Ironborn Pty Ltd

ironborn.com.au <http://ironborn.com.au/>  |  ***@***.***



Getting this email out of normal working hours? I am sending you this email at a time that works for me. I only expect you to respond when convenient for you. #balance


________________________________

**Reply by burkybang** [reply]
It still didn't work. I think you'll have to open GitHub, and upload it through the UI.

**Reply by Serendibb** [reply]
Here you go 
![image](https://github.com/user-attachments/assets/b2861935-5b89-415d-a657-e33e7baf6b00)
![2025-03-11_10h23_37](https://github.com/user-attachments/assets/b01c28a6-f985-4bcb-b15a-10d8905688cb)

**Reply by burkybang** [reply]
Great! That seams to work. Thanks
